### DISCUSSION TITLE
View mode glitch

### DISCUSSION THREAD

**Discussion by mlgarchery** [uxbox/uxbox#1974] [General]
I feel like there is a subtle but annoying glitch in view mode.
The title (text) seems to move between the two artboards. However, it is exactly at the same location on both. Demo:

https://design.penpot.app/#/view/be08c640-e2ca-11ec-9a7f-f940e3befd53?page-id=e2fd9cb1-d6bc-11ec-ab9e-75f6b8214121&section=interactions&index=0&share-id=263356e0-e2cb-11ec-9a7f-f940e3befd53

**Comment by hirunatan** [comment]
Thanks for the reporting. This is caused by the mechanism we use to allow full formatted text inside a svg document. We are aware of this, and are working hard to improve it, but is a very difficult question. We hope to improve this as much as we can in future releases.
